taskw/utils.py

```diff
@@ -134,7 +134,7 @@ def encode_task_experimental(task):
         task[k] = encode_task_value(k, task[k])
 
     # Then, format it as a string
-    return [
+    return [task.pop('description')] + [
```

Comment:
I'm a little confused about this one -- won't this cause us to not encode the task description in the generated task?

Reply:
The diff on this line looks a bit strange but if you look closely you'll see it's removing the description from the `task` dictionary and prepending it as a list item, so it is passed positionaly rather than in `key:value` format.